docs/efiling/efiling_through_docassemble.md

```diff
@@ -0,0 +1,190 @@
+---
+id: efiling_through_docassemble
+title: |
+  E-filing through Docassemble
+sidebar_label: Filing + Docassemble
+slug: /efiling/docassemble
+---
+
+This is a guide to get from a working Assembly Line Interview to an interview that can e-file through our EFSP to the Tyler EFM system.
+
+:::caution
+This e-filing functionality is being updated often, and might be out of date.
+If any thing is wrong or doesn't work as expected, feel free to email us at
+[massaccess@suffolk.edu](mailto:massaccess@suffolk.edu).
+:::
+
+This interview will be simple: making an initial filing into a new case. While our API is flexible enough to
+let you add capabilities and features to your interview, it's also complex, and some of that complexity will need
+to be embedded into your interview.
+
+## Before starting
+
+Before starting this guide, you should have:
+
+* a working docassemble interview, preferably using the AssemblyLine.
+* If you don't have one yet, follow [the steps to author your interview](https://suffolklitlab.org/docassemble-AssemblyLine-documentation/docs/planning).
+* contacted us for access to the EFSP server. We aren't yet making access publicly available, but will include an interest form here when we do.
+
```

Comment:
```suggestion
* Installed the docassemble,EFSP package on your server via the Package Management menu

```

Reply:
I like that step, but I'll wouldn't consider it to be a pre-req, we can walk them through it in this guide.